source/Uncontrolled.js

```diff
@@ -81,7 +81,7 @@ Uncontrolled.propTypes = {
 
 Uncontrolled.defaultProps = {
   closeText: 'Unzoom image',
-  portalEl: (document || {}).body,
+  portalEl: document.body,
```

Comment:
saw in coverage that we weren't testing this. i thought about it, and this just straight wouldn't work in either case, so removing.